lib/sqlite3/pragmas.rb

```diff
@@ -338,6 +338,14 @@ def mmap_size=(size)
       set_int_pragma "mmap_size", size
     end
 
+    def optimize(bitmask = nil)
+      if bitmask
```

Comment:
I'd like to see these bits made available as constants like many of the other sqlite bitmasks in `lib/sqlite3/constants.rb`

Reply:
That's a good idea. I took a pass at this; please let me know what you think.